server/controllers/notification.js

```diff
@@ -0,0 +1,50 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+// createNotification, readNotification, getAll, getUnread
+exports.createNotification = asyncHandler(async (req, res) => {
+    const notification = JSON.parse(req.body)
+    try{
+        notification.save()
+        .then(res.status(200));
+    } catch (error) {
+        res.json(error);
+    }
+});
+exports.readNotification = asyncHandler(async (req, res) => {
+    const { id } = req.params;
```

Comment:
Path params are used to identify/update a single resource. Since we are updating all the notifications below, we should modify this route and rename it accordingly?

Reply:
I am unsure if the changes I made is what you're looking for, please check again; I changed readNotification to readNotifications and modify the patch verb as well as destructure id from req.user now. I only need the user id because this route is used to read all notifications for a single profile and set them to read. I can modify this to take an array of notification id in the patch body if i need this route to do more.